ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/transceiver_qsfp_default_channelized.schema.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  transceiver_qsfp_default_channelized:
+    type: bool
+    default: true
+    description: |
+      The `transceiver_qsfp_default_channelized` knob allows to comply with the change of default QSFP mode for Tomahawk and Trident based devices (40G instead of 4x10G).
+      This change has been introduced starting from EOS version 4.26.0F.
+      The user should not configure the knob to `false` for other type of platforms or for a lower EOS version.
```

Comment:
So I think there is more to this than this reason, since newer EOS versions have removed the command from defaults on other platforms. I think we need to document this with broader terms.
Let's discuss internally.

Reply:
The description has been updated inspired from the EOS guide: https://www.arista.com/en/um-eos/eos-ethernet-ports